pkg/secrets/repositoryserver/secrets_interface.go

```diff
@@ -0,0 +1,5 @@
+package repositoryserver
+
+type RepositoryServerSecrets interface {
+	ValidateSecret() error
```

Comment:
```suggestion
	Validate() error
```
It could be Validate() only?